### ISSUE TITLE
Look at Aalto for xs4s

### ISSUE THREAD

**Issue by ScalaWilliam** [ScalaWilliam/xs4s#6]
https://github.com/FasterXML/aalto-xml

**Comment by rossabaker** [comment]
I'm looking at improving http4s' XML support.  http4s is built on streams (scalaz-stream now, fs2 in the future).  I would like to be able to process XML payloads without blocking (so no InputStream with SAX) or buffering the entire payload as a String before parsing.  I think we may have similar needs here.

https://github.com/http4s/http4s/pull/735

**Comment by ScalaWilliam** [comment]
yes looks very related. not entirely sure about aalto characteristics just yet.

**Comment by ScalaWilliam** [comment]
@rossabaker just want to understand, was this resolved for you in the end on http4s? how does http4s deal with XML currently?

**Comment by rossabaker** [comment]
We still use scala-xml.  As far as I know, Aalto is still the closest we can get to a jawn-like experience with XML, but there just hasn't been the demand for it.
